### PR TITLE
Vault documentation: updated install and ha files

### PR DESCRIPTION
**Pull request by taoism4504**
This is the second batch of changes as part of the efforts to clean up content for Vault documentation ([Asana](https://app.asana.com/0/563192436488770/1202671678968862/f))

The following files were edited and updated:

- Installing Vault :mag: [Deploy Preview](https://vault-git-docs-content-update-batch2-hashicorp.vercel.app/docs/install)
- High Availability :mag: [Deploy Preview](https://vault-git-docs-content-update-batch2-hashicorp.vercel.app/docs/internals/high-availability)

### CODE REVIEW

File: website/content/docs/internals/high-availability.mdx [hunk 1]
```diff
@@ -6,61 +6,48 @@ description: Learn about the high availability design of Vault.
...
-the request will be forwarded onto the active server. If the request is
-read-only the request will be serviced locally on the Performance Standby.
+If a request comes into a Performance Standby Node that causes a storage to write
+the request, the request is forwarded to the active server. Read-only requests are serviced locally on the Performance Standby.
 
-Just like traditional HA standbys if the active node is sealed, fails, or loses
-network connectivity then a performance standby can take over and become the
-active instance.
+Like traditional HA standbys, a Performance Standby Node becomes the active instance when the active node is sealed, fails, or loses
+network connectivity.
```
Comment (yhyakuna) [inline website/content/docs/internals/high-availability.mdx:53]: Can we add **Tutorial** section? 

```suggestion
Like traditional HA standbys, a Performance Standby Node becomes the active instance when the active node is sealed, fails, or loses
network connectivity.

# Tutorial 

Refer to the following tutorials to learn more.

- [Vault with Integrated Storage Reference Architecture](https://learn.hashicorp.com/tutorials/vault/raft-reference-architecture)
- [Vault HA Cluster with Integrated Storage](https://learn.hashicorp.com/tutorials/vault/raft-storage)
- [Vault High Availability with Consul](https://learn.hashicorp.com/tutorials/vault/ha-with-consul)
- [Performance Standby Nodes](https://learn.hashicorp.com/tutorials/vault/performance-standbys)

```

File: website/content/docs/internals/high-availability.mdx [hunk 2]
```diff
@@ -6,61 +6,48 @@ description: Learn about the high availability design of Vault.
...
-of Vault. You don't need to understand these details to
-effectively use Vault. The details are documented here for
-those who wish to learn about them without having to go
-spelunking through the source code. However, if you're an
-operator of Vault, we recommend learning about the architecture
-due to the importance of Vault in an environment.
+Primarily, Vault is used to manage secrets in production environments. Please note that downtime of the Vault service may affect downstream clients.
+
+~> **Advanced Topic!** This page describes the technical details
+of Vault. The details are documented here for knowledge and familiarity with Vault. A review of this section is not required to use Vault. However, due to the signficance of Vault in an environment, we recommend knowing the architecture for all Vault operators.
```
[the remark this answers is not in the document]
Comment (taoism4504) [reply website/content/docs/internals/high-availability.mdx:14]: Glad you confirmed; I was thinking to remove it.

**Comment by taoism4504** [comment]
Great feedback, thanks!

**Review by yhyakuna — APPROVED** [review]
LGTM

**Pull request merged**